R/designSampleSize.R

```diff
@@ -257,39 +260,104 @@ designSampleSizePlots = function(data) {
         stop ("Invalid input")
     }
     
-    text.size = 1.2	
-    axis.size = 1.3	
-    lab.size = 1.7
-    
+    if(isPlotly) {
+        axis.size <- 10
+        lab.size <- 12 
+        text.size <- 12
+    } else {
+        text.size = 1.2	
+        axis.size = 1.3	
+        lab.size = 1.7
+    }
     if (index == "numSample") {
-        plot(data$desiredFC, data$numSample, 
-             lwd=2, xlab="", ylab="", 
-             cex.axis=axis.size, type="l", xaxt="n")
-        axis(1, at=seq(min(data$desiredFC), max(data$desiredFC), 0.05), 
-             labels=seq(min(data$desiredFC), max(data$desiredFC), 0.05),
-             cex.axis=axis.size)
-        mtext("Desired fold change", 1, line=3.5, cex=lab.size)
-        mtext("Minimal number of biological replicates", 2, line=2.5, cex=lab.size)
-        legend("topright",
-               c(paste("FDR is", unique(data$FDR)),
-                 paste("Statistical power is", unique(data$power))),
-               bty="n",
-               cex=text.size)
+        if(isPlotly) {
+            p <- plot_ly(data, x = ~desiredFC, y = ~numSample, type = 'scatter', mode = 'lines',
+                         line = list(width = 2)) %>%
```

Comment:
Should we refrain from using the `%>%` operator?  I think we'd need to import dplyr / magrittr since this is not an operator that's part of base R

Reply:
Done! Thanks